code/modules/games/saintsandsins_cards.dm

```diff
@@ -0,0 +1,400 @@
+ *
+ *  Suitable for 2-4 players.
+ *
+ *  The above and any changes to it should be listed in game on the deck box description_info.
+ */
+
+/obj/item/deck/saintsandsins
+	name = "\improper Saints and Sins deck box"
+	desc = "A deck box for the hit collectable card game, Saints and Sins. The instructions appear to be listed on the side."
+	icon_state = "deck_box_saintsandsins"
+	description_info = "Players start with 5 spell cards, Heretic card starts in play. Game is over when Heretic card dies. Decks have 30 cards + a Heretic card, with up to 3 duplicate spells. If all cards have been drawn, reshuffle discarded spells. <BR><BR>\
+	Active Heretic actions can only be activated once per round, which are marked by the start of the caster's turn. Passive Heretic effects are always active when their conditions are met. Heretics cannot exceed max health. <BR><BR>\
+	Key spells may only be used on your turn. Reaction spells may be used in reaction to another player's spell, even if it is not targeting you. You may cast up to 2 spells on your turn, and draw 1 card after ending your turn. You may only use one reaction spell per round. <BR><BR>\
+	Damage types: <BR><BR>\
+	Fire | Water | Earth | Wind | Decay | Holy <BR><BR>\
+	Suitable for 2-4 players."
+	decktype = /datum/playingcard/saintsandsins
+	decklimit = 31
+
+	var/list/heretics = list()
+	var/list/spells = list()
+
+/obj/item/pack/saintsandsins
+	name = "\improper Saints and Sins builder pack"
+	desc = "A builder pack for hit collectable card game, Saints and Sins. This one will \
+    help any aspiring deck builder get started."
+	icon_state = "card_pack_saintsandsins"
+	decktype = /datum/playingcard/saintsandsins
+	var/pack_type = CARDPACK_BUILDER
+	var/how_many_cards = 21 /// Boosters give less cards than the builder packs.
+
+
+/obj/item/pack/saintsandsins/Initialize()
+	..()
+	var/list/heretic_weights = build_card_weights_list(pack_type, /decl/sas_card/heretic)
+	var/list/spell_weights = build_card_weights_list(pack_type, /decl/sas_card/spell)
+	var/datum/playingcard/saintsandsins/P
+	var/i
+	var/decl/sas_card/picked_card
+	for(i=0, i<how_many_cards, i++)
+		P = new()
+		if(i == 0) // Just pick one Heretic.
+			picked_card = pickweight(heretic_weights)
+			P.card_icon = "saintsandsins_heretic"
+		else
+			picked_card = pickweight(spell_weights)
+			P.card_icon = "saintsandsins_spell"
+		P.name = picked_card.name
+		P.desc = picked_card.desc
+		P.back_icon = "card_back_saintsandsins"
+		cards += P
+
+/datum/playingcard/saintsandsins
+
+/**
+ * Booster packs
+ * Can have special cards not found in builder pack.
+ */
+
+/obj/item/pack/saintsandsins/booster
+	name = "\improper Saints and Sins booster pack"
+	desc = "A booster pack for hit collectable card game, Saints and Sins. This one is labeled 'Basic Witch Kit'."
+	pack_type = CARDPACK_BASICWITCH
+	how_many_cards = 11
+
+/obj/item/pack/saintsandsins/proc/build_card_weights_list(pack_type, root_type)
```

Comment:
You should probably move this under `/obj/item/pack/saintsandsins/Initialize`, as it's not part of the `/booster` type.